templates/fieldset.html.twig

```diff
@@ -0,0 +1,55 @@
+{#
+/**
+ * @file
+ * Theme override for a fieldset element and its children.
+ *
+ * @todo Remove when https://www.drupal.org/node/3010558 is fixed.
+ *
+ * @see template_preprocess_form_element()
+ */
+#}
+{%
+  set classes = [
+    'js-form-item',
+    'form-item',
+    'js-form-wrapper',
+    'form-wrapper',
+  ]
+%}
+<fieldset{{ attributes.addClass(classes) }}>
+  {%
+    set legend_span_classes = [
+      attributes.hasClass('fieldgroup') and attributes.hasClass('form-composite') ? 'form-item__label' : 'fieldset__label',
+      required ? 'js-form-required',
+      required ? 'form-required',
+      attributes.offsetGet('disabled') ? 'is-disabled',
+    ]
+  %}
+  {%
+    set description_classes = [
+     'form-item__description',
+      attributes.offsetGet('disabled') ? 'is-disabled',
+    ]
+  %}
+  {#  Always wrap fieldset legends in a <span> for CSS positioning. #}
+  <legend{{ legend.attributes.addClass('fieldset__legend') }}>
```

Comment:
We should rename this to `fieldset-legend` since there's no block for fieldset

Reply:
Okay

Reply:
well, since 
* I renamed `fieldset-wrapper` to `fieldset__wrapper` (the fieldset content), 
* I renamed `fieldset-label` to `fieldset__label` (span inside the legend) and 
* I added `fieldset__legend` for `<legend>` tag,

I'd improve this direction instead of doing it back.